src/Objs/Font/FontComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import { connect, Obj } from 'scrivito'
+import { Helmet } from 'react-helmet-async'
+import { isFont } from './FontObjClass'
+
+// FontComponent is rendered via the DesignAdjustments component
+export const FontComponent = connect(
+  function FontComponent({ page }: { page: Obj }) {
+    if (!isFont(page)) return null
+
+    const family = CSS.escape(page.get('family'))
```

Comment:
Good point. No `CSS.escape` is not available in node.

I switch to a different approach: I added more validations for the family-name to avoid having to escape the value later on.